components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -955,9 +967,33 @@ void updateConsumerApplication(OAuthConsumerAppDTO consumerAppDTO, boolean enabl
             oAuthAppDO.setRequestObjectEncryptionMethod(requestObjectEncryptionMethod);
             oAuthAppDO.setRequirePushedAuthorizationRequests(consumerAppDTO.getRequirePushedAuthorizationRequests());
             oAuthAppDO.setSubjectTokenEnabled(consumerAppDTO.isSubjectTokenEnabled());
-            oAuthAppDO.setSubjectTokenExpiryTime(consumerAppDTO.getSubjectTokenExpiryTime());;
+            oAuthAppDO.setSubjectTokenExpiryTime(consumerAppDTO.getSubjectTokenExpiryTime());
+            if (OAuth2Util.isJWTAccessTokenOIDCClaimsSeparationEnabled() &&
```

Comment:
Please add a comment by explaining the if condition. I think here from the if condition, we are checking if the new feature is enabled from the UI and whether the property is already existing in the table.

Reply:
OAuth2Util.isJWTAccessTokenOIDCClaimsSeparationEnabled() check the feature enable for server and other one for app